### PR TITLE
Resolved TOS Order for better user experience

### PR DESCRIPTION
**Pull request by EladKarni**
Made it so that TOS only pops up after you check the "I agree". Unchecking it after it has already been checked will uncheck the bools for privacy and terms requiring the user to go through it again.

Example use: https://imgur.com/MMJnKaO

### CODE REVIEW

File: client/pages/register.js [hunk 1]
```diff
@@ -51,11 +51,13 @@ class Register extends Component {
 
     this._bindMethods([
       'handleChange',
+      'handleTOSChange',
```
Comment (UncleClapton) [inline client/pages/register.js:54]: ah lol I didn't realize this component was so far behind...

Not this needs to be changed now... but just so you know class property methods are bound to the instance by default. (the syntax `methodName = () => {...}`) bindMethods pretty obsolete at this point.

**Review by UncleClapton — APPROVED** [review]
Looks good to me!

File: client/pages/register.js (hunk 1 above)
Comment (EladKarni) [reply client/pages/register.js:54]: So I know arrow functions have the context set automatically, but you had it one way, and i didn't want to change that. I can make that change fairly quickly if needed

Comment (UncleClapton) [reply client/pages/register.js:54]: Nah don't worry about it. We'll tackle it when I come around to cleaning this page up. We have a whole new method for input validation that isn't even implemented on this page yet, and we can do it then

Comment (EladKarni) [reply client/pages/register.js:54]: Sounds good!

**Pull request merged**